lib/babylon-to-espree/toAST.js

```diff
@@ -72,6 +72,15 @@ var astTransformVisitor = {
       delete node.bound;
     }
 
+    if (path.isTypeAlias()) {
+      node.type = "FunctionDeclaration";
+      node.generator = false;
+      node.async = false;
+      node.expression = false;
+      node.params = [];
+      node.body = node.right;
+    }
```

Comment:
Do we have some tests for the babylon-to-espress transform? That is surprising to me that this don't have any side effect?

Reply:
https://github.com/babel/babel-eslint/blob/master/test/babel-eslint.js#L68